app/utils/cloud/index.ts

```diff
@@ -24,7 +24,7 @@ type SyncClientConfig = {
 
 export type SyncClient = {
   get: (key: string) => Promise<string>;
-  set: (key: Object, value: string) => Promise<void>;
+  set: (key: string, value: Object | string) => Promise<void>; // Update the parameter type to accept both object and string
```

Comment:
Note: The GitHub Gist Sync requires an object as a parameter instead of a string. If it is forcefully used as a string, the format will remain as a string even if it is forced to be in JSON format.